tempesta_fw/str.h

```diff
@@ -74,7 +74,7 @@
 #define TFW_STR_CN_SHIFT	TFW_STR_FBITS
 #define __TFW_STR_CN_MAX	(~TFW_STR_FMASK)
 /* Str is compound from many chunks, use indirect table for the chunks. */
-#define __TFW_STR_COMPOUND 	(~((1U << TFW_STR_FBITS) - 1))
+#define __TFW_STR_COMPOUND	16 	
```

Comment:
`16` is very strange. I'd say that we don't need the define any more, but actually we still do need it and need it in original form. Please see comments to `TfwStr` structure definition and `__str_grow_tree()`. Probably @keshonok can comment how to make this better this as well.